Sources/Localization/Resources/ar.lproj/Android.strings

```diff
@@ -181,6 +181,7 @@
 "prefs_use_arabic_summary_on" = "سيتم تحويل التطبيق إلى اللغة العربية";
 "prefs_use_arabic_title" = "الوضع العربي";
 "play" = "تشغيل";
+"repeat" = "كرر";
```

Comment:
The Android.strings files are autogenerated from the Android repo, let's not use it. Let's use Localizable.strings for strings dedicated to iOS only, please.

Reply:
Good to know, thanks!

Reply:
Fixed.